src/actions/pimlico.ts

```diff
@@ -44,8 +44,7 @@ export type GetUserOperationStatusReturnType = PimlicoUserOperationStatus
 /**
  * Returns the live gas prices that you can use to send a user operation.
  *
- * - Docs: [TODO://add link]
- * - Example: [TODO://add link]
+ * - Docs: https://docs.pimlico.io/permissionless/reference/pimlico-bundler-actions/getUserOperationGasPrice
```

Comment:
<img width="529" alt="image" src="https://github.com/pimlicolabs/permissionless.js/assets/5251472/150e3e3f-95cf-4d8c-ac64-d8d53d2bc9c7">
We can use createPimlicoBundlerClient